examples/messengers/telegram/4_buttons.py

```diff
@@ -0,0 +1,131 @@
+# %% [markdown]
+"""
+# 4. Buttons
+
+
+This example shows how to display and hide a basic keyboard in Telegram.
+"""
+
+# %%
+import os
+
+from telebot import types
+
+import dff.script.conditions as cnd
+from dff.script import TRANSITIONS, RESPONSE
+from dff.pipeline import Pipeline
+from dff.messengers.telegram import (
+    PollingTelegramInterface,
+    TelegramMessenger,
+    TelegramUI,
+    TelegramButton,
+    update_processing_service,
+)
+from dff.script.responses import Response
+from dff.utils.testing.common import is_interactive_mode
+
+
+# %%
+# Like Telebot, TelegramMessenger only requires a token to run.
+# However, all parameters from the Telebot class can be passed as keyword arguments.
+messenger = TelegramMessenger(token=os.getenv("TG_BOT_TOKEN", "SOMETOKEN"))
+
+
+# %% [markdown]
+"""
+To display or hide a keyboard, you can utilize the `TelegramUI` class.
+It can be initialized either with the `keyboard` parameter (allows `telebot` keyboards)
+or with the `buttons` parameter (the keyboard will be constructed under the hood).
+
+`TelegramUI` should be passed to the `ui` field
+of the generic `Response` class.
+"""
+
+
+# %%
+script = {
+    "root": {
+        "start": {
+            RESPONSE: Response(text="hi"),
+            TRANSITIONS: {
+                ("general", "native_keyboard"): cnd.true(),
+            },
+        },
+        "fallback": {
+            RESPONSE: Response(text="Finishing test, send /restart command to restart"),
+            TRANSITIONS: {
+                ("general", "native_keyboard"): messenger.cnd.message_handler(
+                    commands=["start", "restart"]
+                )
+            },
+        },
+    },
+    "general": {
+        "native_keyboard": {
+            RESPONSE: Response(
+                **{
```

Comment:
Why we do not show this kind of view:
```
Response(
text="..",
ui = TelegramUI(...),
)
```
It's more clear, i think

Reply:
Done